frontend/src/components/shared/WithItemComponent.tsx

```diff
@@ -78,6 +78,7 @@ const createWithItemComponent = <T extends FMSTrackedModel>(withItem: WithItemFu
 	return WithItemComponentParent
 }
 
+export const withCoordinateComponent = createWithItemComponent<Coordinate>(withCoordinate)
```

Comment:
Just FYI, I will probably remove this file from the project. The idea was to create a component with props that match the 'withItem' functions to make refactoring easier, but I think I'll just use WithItemRenderComponent everywhere.